target/linux/rockchip/patches-5.10/107-rockchip-rk3328-enable-LAN-port-on-OrangePi-R1-Plus.patch

```diff
@@ -0,0 +1,54 @@
+--- a/arch/arm64/boot/dts/rockchip/rk3328-orangepi-r1-plus.dts
++++ b/arch/arm64/boot/dts/rockchip/rk3328-orangepi-r1-plus.dts
+@@ -85,6 +85,18 @@
+ 		regulator-min-microvolt = <5000000>;
+ 		regulator-max-microvolt = <5000000>;
+ 	};
++
++	vdd_5v_lan: vdd-5v-lan-regulator {
```

Comment:
Why do you add this as a separate patch and don't include it in the original patch?

Reply:
I actually don't know how to do this properly. I do intend to send these changes to upstream Linux and U-Boot. So I tried to create patches based on upstream, then create a separate patch for OpenWrt (like the LED tweak).


Is it better to make the patch specifically for OpenWrt  by consolidating the 3 patch files into 1?